bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/codemining/LineHeaderCodeMining.java

```diff
@@ -73,4 +77,26 @@ public LineHeaderCodeMining(Position position, ICodeMiningProvider provider, Con
 		super(position, provider, action);
 	}
 
+	public Point drawMultiLine(GC gc, StyledText textWidget, Color color, int x, int y) {
```

Comment:
Is there any chance we can achieve the same result without creating a new API method? Can this behavior be implemented in `draw` ?

Reply:
could you please take a look at the next change 312b76b?